readme.md

```diff
@@ -1 +1,7 @@
-Empty
\ No newline at end of file
+![American Iron Giant](https://m.media-amazon.com/images/M/MV5BMTQ5NTM4NTY5MV5BMl5BanBnXkFtZTgwOTg2Mzc2NjE@._V1_QL75_UX500_CR0,0,500,281_.jpg)
+
+This repo houses a bot that assists the American Iron Front discord server.
+
+Currently, it:
```

Comment:
add install and run directions

git clone, pip install -r requirements.txt, explain the .env file (add an env template?)
python.py